scripts/generate_cert.py

```
@@ -0,0 +1,40 @@
+        "openssl", "req", "-x509",
+        "-newkey", "rsa:4096",
+        "-nodes",
+        "-out", str(cert_path),
+        "-keyout", str(key_path),
+        "-days", "365",
+        "-subj", "/C=US/ST=State/L=City/O=Organization/CN=localhost"
+    ]
+    
+    try:
+        subprocess.run(command, check=True)
+        print(f"SSL certificate and key generated successfully:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)